index.html

```diff
@@ -47,6 +47,154 @@ <h2 id="contacts-heading">LET'S CONNECT</h2>
+                <div class="project-year color-neutrals">2015</div>
+            </section>
+            <img class="project-snapshot" src="./images/project-snapshots/project2.png" alt="project preview">
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="project-languages">
+                <li>html</li>
+                <li>cs</li>
+                <li>javascript</li>
+            </ul>
+            <button type="button" class="button">
+                See Project
+            </button>
+        </section>
+        <section class="project">
+            <h3 class="project-heading">Tonic</h3>
+            <section class="project-info">
+                <div class="project-name">CANOPY</div>
+                <img src="./images/dot_seperator.svg" alt="dot" class="dot-seperator">
+                <div class="project-role color-neutrals">Back End Dev</div>
+                <img src="./images/dot_seperator.svg" alt="dot" class="dot-seperator">
+                <div class="project-year color-neutrals">2015</div>
+            </section>
+            <img class="project-snapshot" src="./images/project-snapshots/project3.png" alt="project preview">
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="project-languages">
+                <li>html</li>
+                <li>cs</li>
+                <li>javascript</li>
+            </ul>
+            <button type="button" class="button">
+                See Project
+            </button>
+        </section>
+        <section class="project">
+            <h3 class="project-heading">Multi-Post
+                Stories</h3>
+            <section class="project-info">
+                <div class="project-name">CANOPY</div>
+                <img src="./images/dot_seperator.svg" alt="dot" class="dot-seperator">
+                <div class="project-role color-neutrals">Back End Dev</div>
+                <img src="./images/dot_seperator.svg" alt="dot" class="dot-seperator">
+                <div class="project-year color-neutrals">2015</div>
+            </section>
+            <img class="project-snapshot" src="./images/project-snapshots/project4.png" alt="project preview">
+            <p class="project-description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="project-languages">
+                <li>html</li>
+                <li>cs</li>
+                <li>javascript</li>
+            </ul>
+            <button type="button" class="button">
+                See Project
+            </button>
+        </section>
+    </section>
+    <section id="about-myself">
+        <h1 id="about-myself-heading">About Myself</h1>
```

Comment:
Nice job with this project you rock it!! 👍 💯 except for having two `h1` tags in your `index.html` file. Since it's recommended to use proper tags in your project but `h1` should be used only once in our project for our website title like you have it here:
```
<h1 id="intro">
   I'm Mr. Mandal <br> Glad to see you!
</h1>
```
Kindly, change the `h1` tag for your `About Myself` and make it `h2` or `h3`.

Reply:
Hi @Mre55, Thank you for your valuable feedback. I have implemented the recommended changes.